Behavioral.Automation.Bindings.UI/Abstractions/ITableWrapper.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Behavioral.Automation.Bindings.UI.Abstractions;
+
+public interface ITableWrapper : IWebElement
+{
+    public Task ShouldBecomeVisibleAsync(int delay);
```

Comment:
If we moved, visiblity binding to IWebElement, why not move this one there, as an overload, as well ?

Reply:
Deleted! Thanks!